src/components/style/modals/dialog.css

```diff
@@ -14,12 +14,15 @@
 }
 
 .dialog-exit {
-  opacity: 1;
   z-index: var(--zindex-shim);
+  opacity: 0;
+  opacity: 1;
```

Comment:
I'm not sure what I was trying to do here but ignore it: I'll fix this tomorrow.